programs/storage_api/src/storage_contract.rs

```diff
@@ -9,14 +9,26 @@ use solana_sdk::hash::Hash;
 use solana_sdk::instruction::InstructionError;
 use solana_sdk::pubkey::Pubkey;
 use solana_sdk::signature::Signature;
+use solana_sdk::syscall::current::Current;
+use solana_sdk::syscall::rewards::Rewards;
 use std::collections::BTreeMap;
 
-pub const VALIDATOR_REWARD: u64 = 200;
-pub const REPLICATOR_REWARD: u64 = 200;
-// Todo Tune this for actual use cases when replicators are feature complete
+// Todo Tune this for actual use cases when PoRep is feature complete
 pub const STORAGE_ACCOUNT_SPACE: u64 = 1024 * 8;
 pub const MAX_PROOFS_PER_SEGMENT: usize = 80;
 
+#[derive(Default, Debug, Serialize, Deserialize, Clone, PartialEq)]
+pub struct Credits {
+    // current epoch
+    epoch: u64,
+    // credits earned in the last epoch
+    pub last_epoch: u64,
```

Comment:
I can't find where last_epoch is used....

Reply:
Unused, I don't know what to do with it right now.

Reply:
delete it, then.

Reply:
I thought I'd need it for something but after the implementation it ended up unused, I'll remove it.